src/modules/languages/python.nix

```diff
@@ -1,21 +1,42 @@
-{ pkgs, config, lib, inputs, ... }:
+{ pkgs, config, lib, ... }:
 
 let
   cfg = config.languages.python;
+  libraries = lib.makeLibraryPath
+    ((lib.optional cfg.manylinux.enable pkgs.pythonManylinuxPackages.manylinux2014Package)
+      # see https://matrix.to/#/!kjdutkOsheZdjqYmqp:nixos.org/$XJ5CO4bKMevYzZq_rrNo64YycknVFJIJTy6hVCJjRlA?via=nixos.org&via=matrix.org&via=nixos.dev
+      ++ [ pkgs.stdenv.cc.cc.lib ]
+      ++ cfg.libraries
+    );
+
+  readlink = "${pkgs.coreutils}/bin/readlink -f ";
+  package = pkgs.callPackage "${pkgs.path}/pkgs/development/interpreters/python/wrapper.nix" {
+    python = cfg.package;
+    requiredPythonModules = cfg.package.pkgs.requiredPythonModules;
+    makeWrapperArgs = [
+      "--prefix"
+      "LD_LIBRARY_PATH"
+      ":"
+      libraries
+    ] ++ lib.optionals pkgs.stdenv.isDarwin [
+      "--prefix"
+      "DYLD_LIBRARY_PATH"
+      ":"
+      libraries
+    ];
+  };
```

Comment:
I think I missed this earlier. Not sure if this is the right way to go, as `LD_LIBRARY_PATH` will seep through to other sub-processes.

That said, I cannot think of a situation where using `DYLD_LIBRARY_PATH` will hurt.

Is it needed to already solve this as part of the CLI rewrite?